src/App.js

```diff
@@ -0,0 +1,33 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import AddCar from './pages/AddCar';
+import DeleteCar from './pages/DeleteCar';
+import Models from './pages/Models';
+import MyReservations from './pages/MyReservations';
+import ReserveCar from './pages/ReserveCar';
+import SignUp from './Components/Authentication/SignUp';
+import LogIn from './Components/Authentication/LogIn';
+import PrivateRoutes from './utils/PrivateRoutes';
+import DetailsPage from './pages/DetailsPage';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this main `App` component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪

Reply:
Hello @Meltrust, thanks for the observation, the file has been updated.